app/jobs/socure_docv_results_job.rb

```diff
@@ -35,9 +43,44 @@ def create_analytics(
     )
   end
 
+  def log_verification_request(docv_result_response:, vendor_request_time_in_ms:)
+    return if docv_result_response.nil?
+
+    response_hash = docv_result_response.to_h
+    verification_response_data = docv_result_response.verification_response_data
+    further_metrics = {
+      user_id: document_capture_session.user.uuid,
+      docv_transaction_token: document_capture_session.socure_docv_transaction_token,
+      submit_attempts: rate_limiter&.attempts,
+      remaining_submit_attempts: rate_limiter&.remaining_count,
+      vendor_request_time_in_ms:,
+      async:,
+      success: response_hash[:success],
+      errors: response_hash[:errors],
+      exception: response_hash[:exception],
+    }
+    @analytics.idv_socure_verification_data_requested(
+      **verification_response_data.to_h.merge(further_metrics),
+    )
+  end
```

Comment:
instead of creating this method ... can we use the existing extra_attributes_method  for all of the attributes alredy in this file and them merge in { vendor_request_time_in_ms: } when making the calls to analytics ... i think it would be moreso inline the pattern of how we are currently handling FormResponses